examples_tests/39.DenoiserTonemapper/main.cpp

```diff
@@ -1024,7 +1018,17 @@ void main()
 					CONVERSION_FILTER convertFilter;
 					CONVERSION_FILTER::state_type state;
 					
-					state.ditherState = _IRR_NEW(std::remove_pointer<decltype(state.ditherState)>::type, ditheringImage);
+					#ifdef BLUE_NOISE_ON_PNG
+					auto ditheringBundle = am->getAsset("../../media/blueNoiseDithering/LDR_RGBA.png", {});
```

Comment:
load dither from EXR... you'll get better dither

Reply:
not an issue anymore - discussed on discord